Makefile

```diff
@@ -14,7 +14,7 @@ $(ODIR)/%.o: $(SDIR)/%.c $(HEADERS) | out
 	$(CC) -c -o $@ $< $(CFLAGS)
 
 $(OUTDIR)/app: $(OBJ) app.c
-	$(CC) -o $@ $^ $(CFLAGS) $(LIBS)
+	$(CC) -o $@ $^ $(CFLAGS) -L/home/travis/build/cjihrig/uvwasi/libuv-v1.32.0/libuv-install/lib $(LIBS)
```

Comment:
I think `LIBRARY_PATH` could be used instead of setting this in the Makefile itself. So perhaps adding it in addition to `LD_LIBRARY_PATH` (which will be needed when dynamically linking at runtime when running ./out/app, vs `LIBRARY_PATH` which is used at link time).

Reply:
I'll give that a shot. I was experimenting with some different things, but was hoping to not need to touch the Makefile at all in the final version of the PR. FWIW, Travis includes this in the logs after installing libuv:

```
Libraries have been installed in:
   /home/travis/build/cjihrig/uvwasi/libuv-v1.32.0/libuv-install/lib

If you ever happen to want to link against installed libraries
in a given directory, LIBDIR, you must either use libtool, and
specify the full pathname of the library, or use the '-LLIBDIR'
flag during linking and do at least one of the following:
   - add LIBDIR to the 'LD_LIBRARY_PATH' environment variable
     during execution
   - add LIBDIR to the 'LD_RUN_PATH' environment variable
     during linking
   - use the '-Wl,-rpath -Wl,LIBDIR' linker flag
   - have your system administrator add LIBDIR to '/etc/ld.so.conf'

See any operating system documentation about shared libraries for
more information, such as the ld(1) and ld.so(8) manual pages.
```

Reply:
> I was experimenting with some different things, but was hoping to not need to touch the Makefile at all in the final version of the PR.

Yeah, the Makefile should not need any changes as far as I can tell, as these variables could be added to the environment or specified beforefor the make command:
```console
$ LIBRARY_PATH="/home/travis/build/cjihrig/uvwasi/libuv-v1.32.0/libuv-install/lib" make
```

Regarding the log I think these are consistent with adding the `LIBRARY_PATH` which should equivalent to using `-LLIBDIR`. And during execution the library should be found thanks to `LD_LIBRARY_PATH` that is already being set. At least that is my take on it.